go-apps/meep-metrics-engine/api/swagger-v2.yaml

```diff
@@ -0,0 +1,790 @@
+        description: "The URL of your own listener application."
+    description: "Notification callback definition."
+    example: "{\"notifyURL\":\"http://clientApp.example.com/metric_event_notifications/123456\"\
+      }"
+  NetworkSubscriptionParams:
+    type: "object"
+    properties:
+      clientCorrelator:
+        type: "string"
+        example: "83"
+        description: "Uniquely identifies this create subscription request. If there\
+          \ is a communication failure during the request, using the same clientCorrelator\
+          \ when retrying the request allows the operator to avoid creating a duplicate\
+          \ subscription."
+      callbackReference:
+        $ref: "#/definitions/Network_callbackReference"
+      networkQueryParams:
+        $ref: "#/definitions/NetworkQueryParams"
+      recurrence:
+        type: "integer"
+        example: 1
+        description: "Recurrence of recurring-time-based notifications"
+      subscriptionType:
+        type: "string"
+        example: "Recurring-time-based"
+        description: "Type of subscription triggering notifications"
+        enum:
+        - "Recurring-time-based"
+    description: "Network metrics subscription parameters"
+    example:
+      clientCorrelator: "0123"
+      callbackReference:
+        notifyURL: "http://clientApp.example.com/natwork_event_notifications/123456"
+      networkQueryParams:
+        tags:
+        - name: "src"
+          value: "ue1-iperf"
+        fields:
+        - "lat"
+        - "tput"
+        - "loss"
+        scope:
+          limit: 60
+      recurrence: 1
+      subscriptionType: "Recurring-time-based"
+  NetworkSubscriptionResponse:
+    type: "object"
+    properties:
+      subscriptionId:
+        type: "string"
+        example: "1.2556487E7"
+        description: "Subscription identifier"
+      clientCorrelator:
+        type: "string"
+        example: "83"
+        description: "Uniquely identifies this create subscription request. If there\
+          \ is a communication failure during the request, using the same clientCorrelator\
+          \ when retrying the request allows the operator to avoid creating a duplicate\
+          \ subscription."
+      callbackReference:
+        $ref: "#/definitions/Network_callbackReference"
+      resourceURL:
+        type: "string"
+        format: "uri"
+        example: "http://localhost:8291/v2/subscriptions/network/subscription123"
+        description: "Self referring URL."
+      networkQueryParams:
+        $ref: "#/definitions/NetworkQueryParams"
+      recurrence:
+        type: "integer"
+        example: 1
+        description: "Recurrence of recurring-time-based notifications"
+      subscriptionType:
+        type: "string"
+        example: "Recurring-time-based"
+        description: "Type of subscription triggering notifications"
+        enum:
+        - "Recurring-time-based"
```

Comment:
Rename property: updateInterval

Description: "Notification interval in seconds, disabled if set to 0"

Reply:
fixed